setup.cfg

```diff
@@ -43,6 +43,8 @@ docs =
     Sphinx
 azure =
     ansible[azure]
+digitalocean =
+    dopy
```

Comment:
```suggestion
    dopy python_version<"3.0"
```

Or?

Reply:
I've updated this for, I'll look to follow up with a seperate PR at a later date to address the digitalocean module changes.  fwiw the requirement on dopy is removed with the new module(s), instead they query the api directly.